torch_xla/experimental/spmd_fully_sharded_data_parallel.py

```diff
@@ -94,7 +95,9 @@ def __init__(
       )
       self._auto_wrap(auto_wrap_kwargs, fsdp_kwargs)
 
-    self._orig_module = module
+    # Let's move the module to xla device in case it's not moved
+    # by the caller already.
+    self._orig_module = module.to(xm.xla_device())
```

Comment:
Is this a no-op if the model is already on xla?

Reply:
I think so.